src/app/hooks/use-search-parameters-validation.ts

```diff
@@ -86,8 +87,18 @@ export const useSearchParametersValidation = () => {
 
 		await env.delegates().sync(profile, network.coin(), network.id());
 
-		if (!delegateFromSearchParameters({ env, network, profile, searchParameters: parameters })) {
-			throw new Error(t("TRANSACTION.VALIDATION.DELEGATE_NOT_FOUND", { delegate: delegateName || publicKey }));
+		const delegate = delegateFromSearchParameters({ env, network, profile, searchParameters: parameters });
+
+		const truncatedName = truncate(delegateName || (publicKey as string), {
```

Comment:
should we truncate only if the value is longer than 20 chars, i.e. only when a public key is used?

Reply:
Fixed @dated, thanks :+1: